vendor/swaggen/models/dependencies_list.go

```diff
@@ -15,7 +15,7 @@ import (
 type DependenciesList struct {
```

Comment:
Autogeneration is not part of the build process, so autogenerated code is stored like this for now